src/plugins/kibana_usage_collection/server/collectors/application_usage/schema.ts

```diff
@@ -124,7 +124,7 @@ export const applicationUsageSchema = {
   kibana: commonSchema, // It's a forward app so we'll likely never report it
   management: commonSchema,
   short_url_redirect: commonSchema, // It's a forward app so we'll likely never report it
-  visualize: commonSchema,
```

Comment:
Let’s not change telemetry keys - the application we are tracking here is still “visualize”, everything else will cause problems